src/modules/shared/components/MediaCard/MediaCard.tsx

```diff
@@ -118,6 +111,12 @@ const MediaCard: FC<MediaCardProps> = ({
 		setIsRequestAccessBladeOpen(true);
 	};
 
+	const saveScrollPosition = () => {
+		if (id && previousPage) {
+			dispatch(setLastScrollPosition({ itemId: id, page: previousPage }));
+		}
+	};
```

Comment:
OPTIONAL: if you add this back in the same location as it was before, then git will not see this as a change. It might be easier to find changes to this code later if we ever need to
low prio though.